### PR TITLE
Cleanup tips section

### PR DESCRIPTION
**Pull request by Moshe-Immerman**
* consolidate cheatsheets
* consolidate blogs
* remove articles about tools already included
* remove presentations 
* remove overly specific tutorials on node, java, CIFS
* remove How to whale as it is no longer working
* remove obsolete NSEnter article
* remove articles listed in nkode's valuable docker links and/or submitted via https://github.com/nkratzke/nkratzke.github.io/pull/1

### CODE REVIEW

**Comment by Moshe-Immerman** [comment]
Sorry about the size of this PR..

File: README.md [hunk 1]
```diff
@@ -87,20 +87,17 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
...
-* [How to Whale](https://github.com/carolynvs/howtowhale) - Learn Docker in your web browser, no setup or installation required by [@carolynvs](https://github.com/carolynvs).
 * [Katacoda](https://www.katacoda.com/): Learn Docker using Interactive Browser-Based Labs
 * [Learn Docker](https://github.com/dwyl/learn-docker) Full environment set up, screenshots, step-by-step tutorial and more resources (video, articles, cheat sheets) by [@dwyl](https://github.com/dwyl)
 * [Play With Docker](http://training.play-with-docker.com/) - PWD is a great way to get started with Docker from beginner to advanced users. Docker runs directly in your browser.
 * [Play With Moby](http://play-with-moby.com/) - PWM is a web based Moby playground which allows you to try different components of the platform in seconds. It gives you the experience of having a free Alpine Linux Virtual Machine in the cloud where you can build and run Moby projects and even create clusters to experiment.
+* **Cheatsheets** by * [@wsargent][docker-cheat-sheet] [@JensPiegsa](http://docker.jens-piegsa.com) [@eon01](https://github.com/eon01/DockerCheatSheet) [@dimonomid][docker-quick-ref] (PDF)
```
Comment (veggiemonk) [inline README.md]: Should be put in a list like so:
* [@wsargent][docker-cheat-sheet] 
* [@JensPiegsa](http://docker.jens-piegsa.com) 
* [@eon01](https://github.com/eon01/DockerCheatSheet) 
* [@dimonomid][docker-quick-ref] (PDF)

File: README.md [hunk 2]
```diff
@@ -544,25 +541,10 @@ Services to securely store your Docker images.
...
-* [Blog](http://jasonwilder.com/) of [@jwilder][jwilder]
-* [Blog](http://jpetazzo.github.io/) of [@jpetazzo][jpetazzo]
-* [Blog](http://progrium.com/blog/) of [@progrium][progrium]
-* [Blog](http://sebgoa.blogspot.be/) of [@sebgoa][sebgoa]
-* [Blog](https://blog.codeship.com/) of [@codeship](https://github.com/codeship)
-* [Blog](https://blog.jessfraz.com/) of [@jessfraz][jessfraz]
-* [Container solutions](http://container-solutions.com/blog/)
-* [Container42](http://container42.com/)
+* __[Valuable Docker Links](http://www.nkode.io/2014/08/24/valuable-docker-links.html)__ High quality articles about docker! __MUST SEE__ 
+* Blogs [by @nickjanetakis](https://nickjanetakis.com/blog/tag/docker) [@crosbymichael](http://crosbymichael.com/) [@gliderlabs](https://gliderlabs.com/devlog/) [@jwilder](http://jasonwilder.com/) [@jpetazzo](http://jpetazzo.github.io/) [@progrium](http://progrium.com/blog/) [@sebgoa](http://sebgoa.blogspot.be/) [@codeship](https://blog.codeship.com/) [@jessfraz](https://blog.jessfraz.com/) * [@Container42](http://container42.com/) [Container solutions](http://container-solutions.com/blog/)
```
Comment (veggiemonk) [inline README.md]: should be listed like so:
* [@nickjanetakis](https://nickjanetakis.com/blog/tag/docker) 
* [@crosbymichael](http://crosbymichael.com/) 
* [@gliderlabs](https://gliderlabs.com/devlog/) 
* [@jwilder](http://jasonwilder.com/) 
* [@jpetazzo](http://jpetazzo.github.io/) 
* [@progrium](http://progrium.com/blog/) 
* [@sebgoa](http://sebgoa.blogspot.be/) 
* [@codeship](https://blog.codeship.com/) 
* [@jessfraz](https://blog.jessfraz.com/) 
* [@Container42](http://container42.com/) 
* [Container solutions](http://container-solutions.com/blog/)

File: README.md (hunk 1 above)
Comment (veggiemonk) [reply README.md]: @Moshe-Immerman I think you forgot something here 😄

**Pull request merged**